woocommerce/woocommerce-gateway-stripe
# Add test mode notice

Fixes #2309 

## Changes proposed in this Pull Request:
- Added a notice in the admin dashboard to make it visible that `test mode` is enabled.
- The approach is similar to [wcpay](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/admin/class-wc-payments-admin-settings.php#L46-L63).

![test mode notice](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/deb39fe4-67fc-4c38-a8d7-c9993f13e177)


## Testing instructions
- Go to **WooCommerce > Settings > Payments > Stripe > Settings**
- Enable Stripe in `test mode`.
- The `test mode` notice should appear on the top. (You may need to refresh. This is expected for all the existing notices in `includes/admin/class-wc-stripe-admin-notices.php` also)
